test/src/test/java/org/corfudb/runtime/collections/SMRMapTest.java

```diff
@@ -734,6 +739,45 @@ public void bulkReads()
                 (endTime - startTime) / MILLISECONDS_TO_MICROSECONDS);
     }
 
+    /**
+     *  test proper backpointer termination at the head of the stream\
+     *
+     * */
+    @Test
+    public void headOfStreamBackpointerTermination() {
```

Comment:
This is a great test. Please can you describe the steps and what and how you are trying to test. It is important for tests to be high quality and have quality reviews. Thanks!